components/dashboard/src/projects/NewProject.tsx

```diff
@@ -217,15 +217,17 @@ export default function NewProject() {
                             <div key={`repo-${r.name}`} className="flex p-3 rounded-xl hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light transition ease-in-out group" >
 
                                 <div className="flex-grow">
-                                    <div className="text-base text-gray-900 dark:text-gray-50 font-medium rounded-xl whitespace-nowrap">{toSimpleName(r.name)}</div>
+                                    <div className={"text-base dark:text-gray-50 font-medium rounded-xl whitespace-nowrap" + (r.inUse ? " text-gray-400" : "text-gray-700")}>{toSimpleName(r.name)}</div>
                                     <p>Updated {moment(r.updatedAt).fromNow()}</p>
                                 </div>
                                     <div className="flex justify-end">
-                                        <div className="h-full my-auto flex self-center opacity-0 group-hover:opacity-100">
+                                        <div className="h-full my-auto flex self-center opacity-0 group-hover:opacity-100 items-center mr-2 text-right">
                                             {!r.inUse ? (
                                                 <button className="primary" onClick={() => setSelectedRepo(r.name)}>Select</button>
                                             ) : (
-                                                <p className="my-auto">already taken</p>
+                                                <p className="text-gray-500 font-medium">
+                                                    @username added<br/ >this in a team
```

Comment:
**issue(blocking)**: The username here needs to be dynamically fetched. ⚠️

Reply:
added that in another commit

Reply:
Cc @jankeromnes because of the [relevant discussion](https://gitpod.slack.com/archives/C021Q4BL8E8/p1628689596191900?thread_ts=1628583510.156800&cid=C021Q4BL8E8) (internal).